packages/chain-events/services/ChainEventsConsumer/republishMessages.ts

```diff
@@ -1,52 +0,0 @@
-import type {
```

Comment:
Any reason this was deleted?

Reply:
Unnecessary. The main reason for this was as a fallback in case of the rare case that a message was not properly queued but since chain-events went live months ago we haven't had a single such case. The base class still exists though so if we ever need to reintroduce it in the future we can.